Update menu.js

rt-2 opened the pull request:
Added the `DefaultIndex` function I made for myself, quite useful if you want to display let say a scale from -1.0 to 1.0 and have the default value to 0.0
rt-2

rt-2 committed:
Update menu.js

Added the `DefaultIndex` function I made for myself, quite useful if you want to display let say a scale from -1.0 to 1.0 and have the default value to 0.0
rt-2

romgerman commented on js/menu.js:
```diff
@@ -205,6 +205,15 @@ class MenuItem {
 
+	DefaultIndex(index) {
+
+		if (index == -1) index = value.length - 1
```
Make a check if `value` is an array first. Then better use `index < 0` instead so the index will not fall into negative value.

rt-2 replied:
https://github.com/romgerman/GTAVMenuVueHtml/pull/8/commits/8616ff03c6983f0ebce648d1374ec2ed1d9d7586

rt-2 committed:
Update menu.js

fixed requested changes and added functionality for negative indexes like -2 would be the second last item.

rt-2 commented:
I don't see what changes you want me to make sorry.
Is it ok or is there still things you want me to edit.
Thank you,
rt-2
